README.md

```diff
@@ -24,20 +26,28 @@ Data collection is easy. Decision making is hard. H2O makes it fast and easy to
 <a name="Downloading"></a>
 ## 1. Downloading H2O-3
 
-While most of this README is written for developers who do their own builds, most H2O users just download and use a pre-built version.  If that's you, just follow these steps:
+While most of this README is written for developers who do their own builds, most H2O users just download and use a pre-built version.  If you are a Python or R user, the easiest way to install H2O is via [PyPI](https://pypi.python.org/pypi/h2o) or [Anaconda](https://anaconda.org/anaconda/h2o) (for Python) or [CRAN](https://CRAN.R-project.org/package=h2o) (for R):  
```

Comment:
That Anaconda link is stale. It should point to the following:
https://anaconda.org/h2oai/h2o 
The above link has the most recent files.